doc/ipythonconsole.rst

```diff
@@ -2,91 +2,159 @@
 
-   You can get :file:`{jupyter/runtime/dir/path}` by executing ``jupyter --runtime-dir`` in the same Python environment as the kernel.
+   .. image:: images/console/console-connect-local-step2.gif
+      :alt: Copying the connection filename into Spyder's dialog
 
-#. Click :guilabel:`Connect to an existing kernel` from the :guilabel:`Console` menu or the :guilabel:`IPython Console` pane's "Gear" menu.
+#. Click :guilabel:`OK` to connect to the kernel.
 
-#. Browse for or enter the path to the connection file from the previous step.
-   If you're connecting to a local kernel, click :guilabel:`Ok` and Spyder should connect to the kernel; if a remote kernel, proceed to the final step.
+   .. image:: images/console/console-connect-local-step3.gif
+      :alt: Connecting to the kernel and running basic commands.
 
-   As a convenience, kernel ID numbers (e.g. ``1234``) entered in the connection file path field will be expanded to :file:`{jupyter/runtime/dir/path}/kernal-{id}.json` on your local machine.
 
-#. If connecting to a remote kernel over ``ssh``, check the appropriate box and type the full hostname you're connecting to (in the form :file:`{username}@{hostname}:{port-number}`).
-   Then, enter *either* :file:`{username}` 's password on the remote machine, or your user SSH keyfile (typically :file:`.perm`) (only one is needed to connect), and press :guilabel:`Ok`.
+Connect to a remote kernel
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   The port number is the one on which the SSH daemon (``sshd``) is running, typically 22 unless you or your administrator has configured it otherwise.
+To connect to a kernel on a remote machine,
 
-.. image:: images/console/console-dialog-connect.png
-   :alt: Connect to kernel dialog, requesting path and connection details
+#. Launch a Spyder kernel on the remote host if one is not already running, with ``python -m spyder_kernels.console``.
 
-For more technical details about connecting to remote IPython kernels, see the `Connecting to a remote kernel`_ page in the IPython Cookbook.
-Just remember to enter the appropriate details into Spyder's :guilabel:`Connect to an existing kernel` dialog instead of launching a new frontend on the client with ``--existing``.
+   .. image:: images/console/console-connect-remote-step1.gif
+      :alt: Staring a Spyder kernel on a remote machine
 
-.. _Connecting to a remote kernel: https://github.com/ipython/ipython/wiki/Cookbook:-Connecting-to-a-remote-kernel-via-ssh
+#. Copy the kernel's connection file (:file:`{jupyter/runtime/dir/path}/kernel-{pid}.json`) to the machine you're running Spyder on.
 
+   You can get :file:`{jupyter/runtime/dir/path}` by executing ``jupyter --runtime-dir`` in the same Python environment as the kernel.
+   Usually, the connection file you are looking for will be one of the newest in this directory, corresponding to the time you started the external kernel.
 
-.. _console-features:
+   .. image:: images/console/console-connect-remote-step2.gif
```

Comment:
- I also suggest to use `conda activate` instead of `source` for this one.
- Also, if you don't agree with using `nohup`, I think it's better for you to select the terminal where you started the kernel with your mouse, to give people some time to realize the kernel lives in a different terminal (it took me some time to do that).

Reply:
>     * I also suggest to use `conda activate` instead of `source` for this one.

See previous response; its a `venv`, not a `conda` env.

>     * Also, if you don't agree with using `nohup`, I think it's better for you to select the terminal where you started the kernel with your mouse, to give people some time to realize the kernel lives in a different terminal (it took me some time to do that).

Yeah, I agree this could be clearer; I'm just so used to Ctrl-Tabbing between console tabs that I didn't think of that till after I recorded the GIF. I can get everything set up and re-record it then.

Reply:
Can you slow down this one for a little bit and maybe add a longer pause at the end? It feels like everything is happening a little bit too fast and it's hard to read what's happening.

Reply:
I didn't want to make the GIF too long, since its already pretty lengthy at 14 s, but I decreased it from 4x to 3x, and (coupled with the slowdown) and increased the pause at the end from a little over 1 s to close to 2.5 s. It's on the long side, but still just under 20 s, while hand-tuning ensured a minimal increase in file size.